RecoLocalFastTime/FTLCommonAlgos/plugins/FTLSimpleUncalibRecHitAlgo.cc

```diff
@@ -24,14 +24,16 @@ class FTLSimpleUncalibRecHitAlgo : public FTLUncalibratedRecHitAlgoBase {
   void getEventSetup(const edm::EventSetup&) final {}
 
   /// make the rec hit
-  FTLUncalibratedRecHit makeRecHit(const FTLDataFrame& dataFrame ) const final;
+  //FTLUncalibratedRecHit makeRecHit(const FTLDataFrame& dataFrame ) const final;
```

Comment:
remove commented-out code
(or keep it and uncomment it so D19 doesn't break...)